libnymea/nymeasettings.cpp

```diff
@@ -104,6 +104,9 @@ NymeaSettings::NymeaSettings(const SettingsRole &role, QObject *parent):
     case SettingsRoleTags:
         fileName = "tags.conf";
         break;
+    case SettingsRoleMqttPolicies:
+        fileName = "mqttpolicies.conf";
+        break;
```

Comment:
Please add these Settings also to the debug server for downloading and for the report generator.